python/mlc_chat/quantization/per_tensor_quantization.py

```diff
@@ -176,7 +176,18 @@ def _quantize_float8(self, weight: nn.Tensor):
         quantize_dtype = DataType(self.weight_dtype)
 
         if self.no_scale:
-            scaled_weight = weight.astype(self.weight_dtype)
+            # TODO(csullivan, vinx13): Ensure scheduling is applied after lowering relax ops.
+            # Currently, dlight scheduling is applied before the R.cast is lowered.
+            # scaled_weight = weight.astype(self.weight_dtype)
+            scaled_weight = nn.tensor_expr_op(
```

Comment:
Currently we get a error during slm/build.py for convert-weights-only due to the fact that the cast op is not scheduled for GPU (standard memory access verification failure). The reasoning is that dlight scheduling runs before the R.cast is lowered to a primfunc. Alternatively we could update the pipeline used during compiling the quantization functions so that dlight scheduling would also occur after the R.cast is lowered. That approach is beyond my current number of cycles.